applications/RomApplication/python_scripts/rom_analysis.py

```diff
@@ -110,26 +111,47 @@ def ModifyAfterSolverInitialize(self):
-            aux = KratosMultiphysics.Matrix(nodal_dofs, rom_dofs)
-            for node in computing_model_part.Nodes:
-                node_id = str(node.Id)
-                for j in range(nodal_dofs):
-                    for i in range(rom_dofs):
-                        aux[j,i] = nodal_modes[node_id][j][i].GetDouble()
-                node.SetValue(KratosROM.ROM_BASIS, aux)
-            
-            # Set the left nodal ROM basis if it is different than the right nodal ROM basis (i.e. Petrov-Galerkin)
-            if (self.solving_strategy == "petrov_galerkin"):
-                petrov_galerkin_nodal_modes = self.rom_parameters["petrov_galerkin_nodal_modes"]
-                petrov_galerkin_nodal_dofs = len(self.project_parameters["solver_settings"]["rom_settings"]["nodal_unknowns"].GetStringArray())
-                aux = KratosMultiphysics.Matrix(petrov_galerkin_nodal_dofs, self.petrov_galerkin_rom_dofs)
+            # Set the right nodal ROM basis
+            if self.rom_parameters["rom_format"].GetString() == "json":
+                aux = KratosMultiphysics.Matrix(nodal_dofs, rom_dofs)
                 for node in computing_model_part.Nodes:
                     node_id = str(node.Id)
-                    for j in range(petrov_galerkin_nodal_dofs):
-                        for i in range(self.petrov_galerkin_rom_dofs):
-                            aux[j,i] = petrov_galerkin_nodal_modes[node_id][j][i].GetDouble()
-                    node.SetValue(KratosROM.ROM_LEFT_BASIS, aux)
+                    for j in range(nodal_dofs):
+                        for i in range(rom_dofs):
+                            aux[j,i] = nodal_modes[node_id][j][i].GetDouble()
+                    node.SetValue(KratosROM.ROM_BASIS, aux)
+
+                # Set the left nodal ROM basis if it is different than the right nodal ROM basis (i.e. Petrov-Galerkin)
+                if (self.solving_strategy == "petrov_galerkin"):
+                    petrov_galerkin_nodal_modes = self.rom_parameters["petrov_galerkin_nodal_modes"]
+                    petrov_galerkin_nodal_dofs = len(self.project_parameters["solver_settings"]["rom_settings"]["nodal_unknowns"].GetStringArray())
+                    aux = KratosMultiphysics.Matrix(petrov_galerkin_nodal_dofs, self.petrov_galerkin_rom_dofs)
+                    for node in computing_model_part.Nodes:
+                        node_id = str(node.Id)
+                        for j in range(petrov_galerkin_nodal_dofs):
+                            for i in range(self.petrov_galerkin_rom_dofs):
+                                aux[j,i] = petrov_galerkin_nodal_modes[node_id][j][i].GetDouble()
+                        node.SetValue(KratosROM.ROM_LEFT_BASIS, aux)
+
+            elif self.rom_parameters["rom_format"].GetString() == "numpy":
+                #### Set the nodal ROM basis ####
+                Nodes = np.load("NodeIds.npy")
+                RightModes = np.load("RightBasisMatrix.npy")
```

Comment:
```suggestion
                nodes = np.load("NodeIds.npy")
                right_modes = np.load("RightBasisMatrix.npy")
```
Please stick to our [style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide). In this case, note that a local variable is not written in `snake_case` as it should be but in `CamelCase`.